ld[34mTest run [0mscala.scalanative.junit.[33mExceptionAfterAssume[0m[34m started[0m
ldTest scala.scalanative.junit.[33mExceptionAfterAssume[0m.[36massumeFail[0m started
leTest scala.scalanative.junit.[33mExceptionAfterAssume[0m.[31massumeFail[0m failed: org.junit.[31mTestCouldNotBeSkippedException[0m: Test could not be skipped due to other failures, took <TIME>
leCaused by: org.junit.AssumptionViolatedException: This assume should not pass
e2scala.scalanative.junit.ExceptionAfterAssume.assumeFail
leTest scala.scalanative.junit.[33mExceptionAfterAssume[0m.[31massumeFail[0m failed: java.lang.[31mIllegalArgumentException[0m: after() must be called, took <TIME>
ldTest scala.scalanative.junit.[33mExceptionAfterAssume[0m.[36massumeFail[0m finished, took <TIME>
li[34mTest run [0mscala.scalanative.junit.[33mExceptionAfterAssume[0m[34m finished: [0m[31m2 failed[0m[34m, [0m[34m0 ignored[0m[34m, 1 total, <TIME>[0m
d
